tedge_config/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tedge_config"
+version = "0.1.0"
+authors = ["Software AG <thin-edge-team@softwareag.com>"]
+edition = "2018"
+
+[dependencies]
+toml = "0.5"
```

Comment:
Good point. Finally it can be removed. Currently, we still have two TEdgeConfig implementations at the same time, so we still need it. Once we migrate `tedge` to use `tedge_config`, we should be able to get rid of it.